src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperBase;
+    private double lowerBase;
+
+    public IsoscelesTrapezoid(double upperBase,double lowerBase, Color color) {
+        super(color);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((upperBase + lowerBase) / 2 * calculateHeight());
+    }
+
+    public double calculateHeight() {
+        return Math.sqrt(upperBase + lowerBase);
```

Comment:
The formula used in `calculateHeight` is incorrect for calculating the height of an isosceles trapezoid. The height should be calculated using the formula: `sqrt(side^2 - ((lowerBase - upperBase) / 2)^2)`, where `side` is the length of the non-parallel sides. Ensure you have the correct parameters and logic for this calculation.